### DISCUSSION TITLE
Can you implement a fix to notion's recent stupid feature which kills kanban boards?

### DISCUSSION THREAD

**Discussion by nandonakis** [GorvGoyl/Notion-Boost-browser-extension#174] [Ideas]
More info here.
https://www.reddit.com/r/Notion/comments/11obg6u/recent_issue_with_dragging_and_dropping_cards_in/
I'd be looking for an option to disable drag and drop onto the card/page, and make it drop to the board column, as it should.

**Comment by GorvGoyl** [comment]
that is indeed dumb That they thought it'd be a good UX. let me see if it
can be disabled.

**Reply by nandonakis** [reply]
What a legend!

**Reply by nandonakis** [reply]
actually looks like they reverted it lol
